docs/docs/whatsnew.md

```diff
@@ -1,5 +1,30 @@
 # <b>Important updates</b>
 
+## v1.2.25
+
+### Configuration of `maven` builder has changed. 
+As of version 1.2.25, the `mvn -B clean package` command is used where the `maven` builder is configured for building a module or in a global build step. 
+
+The `-B` parameter is added to `maven` builder's build command which will start maven in a interactive mode.
+
+As a build tool, MBT packages all modules’s build process internal, all native builders, such as golang, maven and nodejs, run build command in batch mode is resonable.
```

Comment:
Can you explain this? Not sure I understood.
As a build tool, MBT packages all modules’s build process internal, all native builders, such as golang, maven and nodejs, run build command in batch mode is resonable.

Reply:
MBT builds MTA (Multiple Target Application), MTA contains many kind of modules, each module will be build by native builders, such as golang for go type module, maven for java type module, nodejs for nodejs type module.

MBT is an build tool, when MTA application is build, all build process for different module is packaged into MBT, it wrapped all build process internal and it should not run in an interactive modle, it is reasonable for MBT exec the build process in a batch modle.

Reply:
Ok. Can we use this?

MBT builds the MTA (Multitarget Application), which contains many type of modules. Each module is built by native builders, such as Golang for the Go module, Maven for the Java module, and Node.js for the Node.js module.

When the MTA is built, all build processes for different modules are packaged into the MBT. It wraps all the build processes as internal and it should not run in interactive mode. It is reasonable for MBT to execute the build process in batch mode.

Reply:
OK, I do a little simplification based on your advise

As a build tool, MBT builds the MTA (Multi-Target Application), which contains many type of modules. Each module is built by corresponding builder, such as maven, nodejs and go.

When the MTA is built, all build processes for different modules are packaged into the MBT. It wraps all the build processes as internal and it should not run in interactive mode. It is reasonable for MBT to execute the build process in batch mode.